CleverPush/Source/CPAppBannerButtonBlock.m

```diff
@@ -9,6 +9,7 @@ - (id)initWithJson:(NSDictionary*)json {
         self.type = CPAppBannerBlockTypeButton;
 
         self.text = @"";
+        self.isButtonClicked = false;
```

Comment:
Why do we need this - can't we find an easier solution?

Reply:
That we pass down the button ID in the button click listener. That would be much more clean.